### PR TITLE
PWN-5953 - Replace NoOp -> Unit, small refactoring

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-5953

## Description of Work

NoOp is no more
also extracted storage and room to separate module to make them readable

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/common/ui/widget/CustomSnackBar.kt [hunk 1]
```diff
@@ -40,17 +39,15 @@ class CustomSnackBar @JvmOverloads constructor(
...
+    override fun animateContentOut(delay: Int, duration: Int) = Unit
 
-    fun setMessage(text: String): CustomSnackBar {
+    fun setMessage(text: String): CustomSnackBar = apply {
         binding.textView.text = text
-        return this
     }
 
-    fun setIcon(@DrawableRes iconResId: Int): CustomSnackBar {
+    fun setIcon(@DrawableRes iconResId: Int): CustomSnackBar = apply {
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/common/ui/widget/CustomSnackBar.kt:48]: not obvious at all )

File: app/src/main/java/org/p2p/wallet/qr/ui/ScanQrFragment.kt [hunk 2]
```diff
@@ -81,9 +80,7 @@ class ScanQrFragment :
 
     private val barcodeCallback: ZXingScannerView.ResultHandler =
         ZXingScannerView.ResultHandler { rawResult ->
-            rawResult?.text?.let { address ->
-                validateAddress(address)
-            } ?: showInvalidDataError()
+            rawResult?.text?.let { address -> validateAddress(address) } ?: showInvalidDataError()
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/qr/ui/ScanQrFragment.kt:83]: second mapping operation with new line?

File: app/src/main/java/org/p2p/wallet/updates/SocketUpdatesManager.kt [hunk 3]
```diff
@@ -11,17 +18,11 @@ import kotlinx.coroutines.launch
...
-import org.p2p.wallet.common.di.AppScope
-import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManager
-import org.p2p.wallet.utils.NoOp
-import timber.log.Timber
-import java.util.concurrent.Executors
-import kotlin.properties.Delegates
 
 private const val DELAY_MS = 5000L
 
+private const val TAG = "SOCKET"
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/updates/SocketUpdatesManager.kt:24]: SocketUpdateManager

**Review by Davran331122 — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]
lgtm, gj

**Review by NikolaiEmelianov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/common/ui/widget/CustomSnackBar.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/common/ui/widget/CustomSnackBar.kt:48]: very popular pattern to update instance and return it, i saw it in our codebase

**Comment by Davran331122** [comment]
> lgtm, gj
Absolutely agree with this

**Pull request merged**